mpi-proxy-split/unit-test/Makefile

```diff
@@ -53,7 +53,7 @@ default: ${TEST_BINS}
 ../record-replay.o ../drain_send_recv_packets.o:
 	@make -C ..
 
-%.exe: %.o ../record-replay.o ../split_process.o ../lower-half/procmapsutils.o
+%.exe: %.o ../record-replay.o #../split_process.o ../lower-half/procmapsutils.o
```

Comment:
There's no harm in keeping these objects files. It's compatible with and without the `SINGLE_CART_REORDER` macro. Let's put a Note above this rule that these two objects are needed with the macro, and one should remove these two object files if they choose to remove the single_cart_reorder implementation.